src/org/zaproxy/zap/extension/proxies/OptionsProxiesPanel.java

```diff
@@ -38,6 +39,8 @@
 public class OptionsProxiesPanel extends AbstractParamPanel {
 
     private static final long serialVersionUID = 1L;
+    
+    private static JScrollPane proxiesScrollPane;

```

Comment:
The static can be removed.

Reply:
Whoops yup, that was from an earlier iteration. Sorry